File: 1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let newArr = [];
    for(let i=0; i<nums.length; i++){
        for(let j=i +1; j<nums.length; j++){
            if(nums[i]+nums[j]===target){
                newArr[0] = i;
                newArr[1] = j;
            }
        }     
    }
    return newArr;
};

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map = {}
     
        for i, n in enumerate(nums):
            rem = target - n
            if rem in map:
                return [i, map[rem]]
            else:
                map[n] = i